cns/restserver/homeazmonitor.go

```diff
@@ -126,6 +126,12 @@ func (h *HomeAzMonitor) Populate(ctx context.Context) {
 				h.update(returnCode, returnMessage, cns.HomeAzResponse{IsSupported: true})
 				return
 
+			case http.StatusNotFound:
+				returnMessage := fmt.Sprintf("[HomeAzMonitor] region does not support AZs probably, failed with StatusCode: %d, error: %v", apiError.StatusCode(), err)
+				returnCode := types.NotFound
```

Comment:
do we use this returnCode to classify this as 'error' and potentially 'retry'?
If yes, we don't want to do treat this as an error. We should still treat it as success where this is equivalent to NMA not supporting homeAZ API

Reply:
Addressed